tests/integration/llm/client.py

```diff
@@ -114,6 +118,24 @@ def test_ds_raw_model(model):
                 assert float(memory) / 1024.0 < spec["max_memory_per_gpu"]
 
 
+def test_sd_handler(model, model_spec):
+    if model not in model_spec:
+        raise ValueError(f"{model} is not one of the supporting models {list(sd_model_spec.keys())}")
+    spec = sd_model_spec[model]
```

Comment:
please also add a worker number check in here

Reply:
good catch